### PR TITLE
Added FAQ item on exposure

### PR DESCRIPTION
**Pull request by glaslos**

### CODE REVIEW

**Comment by vanhoefm** [comment]
What do you mean with "outside of my WiFi network"? If the attacker is within range of a network and client, attacks can be performed.

**Comment by glaslos** [comment]
Maybe I misunderstood, but you still require the target to be on your AP. So as long as I trust my AP and the clients in my WiFi network I'm not exposed.

**Comment by atipaHy** [comment]
@glaslos Did you read the [website ](https://www.krackattacks.com/)and/or see the demo? The **attacker** doesn't need to be **inside** the network. The **attacker** needs to be **within range** of a network and client to perform the attack.

**Comment by glaslos** [comment]
I did see the demo and it's very clear that the target needs to connect to the rogue wifi.

**Comment by DanAtkinson** [comment]
@glaslos I think that you're fundamentally misunderstanding the attack. If you listen to the [following portion of the video](https://youtu.be/Oh4WURZoR98?t=48) (transcript below) it should be evidently clear.

> Notice that I included the protected WiFi network that will be attacked and we will only target one specific device.
> The tool starts by searching for the protected WiFi network we will attack and **then it will clone this network on a different channel.**
> This malicious clone of the network enables the attacker to reliably manipulate handshake messages which is required to abuse the weaknesses we discovered.

So, as per the highlighted quote, the attack comes from **outside** of the network.

**Comment by glaslos** [comment]
The attack creates a rogue access point (I think cloning is a bit misleading here). This is old news. Only when the target establishes a connection to the rogue access point you are able to exploit the WPA2 vulnerability.

**Comment by DanAtkinson** [comment]
@glaslos My point (and everyone elses) is that the attacker **does not** have to be 'on the same WiFi network in order to exploit this vulnerability' as per your PR. You only need to be in range of the victim and the WiFi network.

**Comment by glaslos** [comment]
My point is that the target need to connect to the rogue WiFi, quoting the web page: 
> If the victim is very close to the real network, the script may fail because the victim will always directly communicate with the real network, even if the victim is (forced) on a different Wi-Fi channel than this network.

**Comment by DanAtkinson** [comment]
If that was your point, you did not make it in the PR, and should therefore either close or modify it accordingly because, as it stands right now, it's incorrect.

**Comment by glaslos** [comment]
Okay, let's assume you have successfully exploited the vulnerability. In order to actually sniff the traffic, you need the client to connect to your AP and communicate through it (hence the enabling network forward in the demo). I will adjust my PR to make some of the points you made more clear.

**Comment by glaslos** [comment]
I would really like to see more details on the practicality of the attack. Also some clearer information on the steps required (besides the rough details in the demo).

**Comment by stuntguy3000** [comment]
@glaslos Be noted the correct term is Wi-Fi **not** WiFi

File: index.html [hunk 1]
```diff
@@ -258,6 +258,9 @@ <h2 class="firsttitle">Introduction</h2>
 			<h3 class="firsttitle">Do we now need WPA3?</h3>
 			<p>No, luckily <strong>implementations can be patched in a backwards-compatible manner</strong>. This means a patched client can still communicate with an unpatched access point, and vice versa. In other words, a patched client or access points sends exactly the same handshake messages as before, and at exactly the same moments in time. However, the security updates will assure a key is only installed once, preventing our attacks. So again, update all your devices once security updates are available.</p>
 			
+			<h3>Does this attack work from outside of my WiFi network?</h3>
+			<p>In order to execute the exploit, the attacker is competing with the target WiFi access point. The adversary has to overcome signal strength and MAC spoofing detection. If the attacker successfully spoofs the WiFi access point, executes the exploit, convincing the target to use his access point, the attack would be considered successful.</p>
```
Comment (qwertzguy) [inline index.html:262]: "overcome signal strength": no it doesn't: https://youtu.be/Oh4WURZoR98?t=1m47s (CSA = Channel Switch Announcement)
"MAC spoofing detection": no it doesn't: as long as the person can access at least some endpoints, data can be captured. For example, if the client tries to load a public website, the attack will still be able to capture valuable data even if the attackers network does not proxy to the original network.

**Comment by djdisikd** [comment]
Does anyone try this tool before? Did it works?

**Comment by vanhoefm** [comment]
The attacker is not competing with the target Wi-Fi access point based on signal strength. You can send Channel Switch Announcements that effectively command the victim into switching to the channel where your *clone of the target network* is located. This is based on follow-up research that is not published yet. You should assume it's always possible to carry out the attack against vulnerable devices.

**Comment by vanhoefm** [comment]
So the Q&A answer should be: if you are within range of the client and AP, you can carry out the attack. Special antenna's can be used to carry out the attack from quite far away.

**Comment by vanhoefm** [comment]
Commit 6e2a0d3d12469ed3a73417ead6bdf6abd221c3c2 clarifies how close the attacker needs to be, and that it does not complete with the signal strength of the real network.

**Pull request closed without merging**